apps/elewa-website/src/app/app.module.ts

```diff
@@ -6,10 +6,11 @@ import { FeaturesPagesHomeModule } from '@elewa-website/features/pages/home';
 import { AppComponent } from './app.component';
 import { NxWelcomeComponent } from './nx-welcome.component';
 import { TranslocoRootModule } from './transloco-root.module';
+import { ElementsLayoutButtonsModule } from '@elewa-website/elements/layout/buttons';
```

Comment:
No need to import and showcase the button in the appcomponent. any module/component that will need the button will import it.

Reply:
I have removed the Component from the import